public/app/plugins/datasource/cloudwatch/utils/datalinks.ts

```diff
@@ -70,14 +70,11 @@ function createAwsConsoleLink(
   replace: (target: string, fieldName?: string) => string,
   getVariableValue: (value: string) => string[]
 ) {
-  const arns = target.logGroups?.flatMap((group) => {
-    if (group.value === undefined) {
-      return [];
-    }
-    return [group.value.replace(/:\*$/, '')]; // remove `:*` from end of arn
-  });
+  const arns = (target.logGroups ?? [])
+    .filter((group) => group?.value)
+    .map((group) => (group.value ?? '').replace(/:\*$/, '')); // remove `:*` from end of arn
   const logGroupNames = target.logGroupNames;
```

Comment:
```suggestion
const logGroupNames = target.logGroupNames ?? [];
```
this will fix typings